src/components/network-modification-tree-pane.jsx

```diff
@@ -185,6 +189,24 @@ export const NetworkModificationTreePane = ({ studyUuid, studyMapTreeDisplay })
         }
     }, [broadcastChannel, dispatch, snackInfo]);
 
+    const reorderSubtree = useCallback(
+        (parentNodeId, orderedChildrenNodeIds) => {
+            // We check if the current position is already correct
+            const children = treeModelRef.current.getChildren(parentNodeId);
+            if (orderedChildrenNodeIds.length !== children.length) {
+                snackWarning({
+                    messageId: 'ReorderSubtreeInvalidNotifInfo',
+                });
+                console.warn('Subtree order update cancelled : the size of the ordered children list is incompatible');
+                return;
+            }
```

Comment:
This code is missing the update with your new needReorder function.

Reply:
Yes it'only from a notification !